packages/design-system-docs/src/pages/components/Dropdown/Dropdown.example.jsx

```diff
@@ -12,12 +12,23 @@ const dropdownOptions = [
   { label: 'Option 7', value: '7' },
   { label: 'Option 8', value: '8' },
 ];
+const dropdownOptions2 = [
```

Comment:
Actually, I wonder if we can make every single dropdown use the same options. Reason being is that this extra `dropdownOptions2` is going to show up in the code snippet for this example, and it makes the example code unnecessarily longer.

Reply:
Plus I dont think its a bad idea in general for all our examples to use that placeholder option